_scripts/clone-l10n.sh

```diff
@@ -131,16 +131,28 @@ copy_ftl() {
 PAYMENTS="fxa-payments-server"
 SETTINGS="fxa-settings"
 AUTH="fxa-auth-server"
+REACT="fxa-react"
+SHARED="fxa-shared"
 
-# Copy .ftl files for payments, settings, and auth (emails)
+# Copy .ftl files for payments, settings, auth (emails),
+# react (shared react components), and shared (branding)
 case "$MODULE" in
     "$PAYMENTS")
         copy_ftl "payments"
+        copy_ftl "shared"
         ;;
     "$SETTINGS")
         copy_ftl "settings"
+        copy_ftl "shared"
         ;;
     "$AUTH")
         copy_ftl "auth"
+        copy_ftl "branding"
+        ;;
+    "$REACT")
+        copy_ftl "shared"
```

Comment:
Does `fxa-react` need the l10n repo / the shared file copied into it? Since every package using components out of `fxa-react` should be getting that shared FTL file, I'm not sure that we do.

Reply:
Yeah, I was wondering if we could just point the bundles to the original copy of `fxa-content-server-l10n` (instead of the one copied into each package). It's definitely a lot of duplication.

For now, it looks like `fxa-react`'s storybook isn't set up to show localized messages anyway (like `fxa-settings` - https://github.com/mozilla/fxa/issues/14477). So I guess it depends if we want to add localization support for all SB?

Reply:
Personally I think we should wait to address this (i.e. localizing storybook in fxa-react) until we land FXA-2156. That's just me though. I've been working on CI optimizations and the extra clones do add some unneeded overhead.